notebooks/official/pipelines/google_cloud_pipeline_components_TPU_model_train_upload_deploy.ipynb

```diff
@@ -461,7 +438,9 @@
         "id": "init_aip:mbsdk"
       },
       "source": [
-        "## Initialize Vertex AI SDK for Python\n",
+        "### Initialize Vertex AI SDK for Python\n",
+        "\n",
+        "To get started using Vertex AI, you must [enable the Vertex AI API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com).\n",
```

Comment:
- change: "To get started using Vertex AI, you must [enable the Vertex AI API]"
- to: "To get started using Vertex AI, you must enable the  [Vertex AI API]"